src/views/pages/groups/index.js

```diff
@@ -0,0 +1,18 @@
+// Group Page uses Group1.js, Group2.js, MainContent
+// project imports
+import MainCard from "ui-component/cards/MainCard";
+import Group1 from "./components/Group1";
+import Group2 from "./components/Group2";
+
+// ==============================|| TYPOGRAPHY ||============================== //
+
+const Groups = () => {
+    return (
+        <MainCard>
+            <Group1 />
```

Comment:
Instead of creating 2 different files for each group, use only one file and use properties to define what the group needs to be.  
We get a list of group Id's from the API, so we will need to pass the Id to the component.

Also, please move these changes to the correct Issue & Branch. This is unrelated to FE-12